hack/e2e.sh

```diff
@@ -309,12 +309,7 @@ function test_cassandracluster() {
 
     # Increment the replica count
     export CASS_REPLICAS=2
-    kubectl apply \
-        --namespace "${namespace}" \
-        --filename \
-        <(envsubst \
-              '$NAVIGATOR_IMAGE_REPOSITORY:$NAVIGATOR_IMAGE_TAG:$NAVIGATOR_IMAGE_PULLPOLICY:$CASS_NAME:$CASS_REPLICAS:$CASS_CQL_PORT' \
-              < "${SCRIPT_DIR}/testdata/cass-cluster-test.template.yaml")
+    apply_cassandracluster
```

Comment:
Should we fail the test if this fails? (like we do above)

Reply:
:+1: